test/truefi2/CollateralVault.test.ts

```diff
@@ -0,0 +1,65 @@
+import { expect, use } from 'chai'
+import { solidity } from 'ethereum-waffle'
+import { Wallet } from 'ethers'
+import { setupDeploy } from 'scripts/utils'
+import { beforeEachWithFixture } from 'utils'
+import {
+  CollateralVault,
+  CollateralVault__factory,
+  BorrowingMutex,
+  BorrowingMutex__factory,
+  LineOfCreditAgency,
+  LineOfCreditAgency__factory,
+  Liquidator2,
+  Liquidator2__factory,
+  TrustToken,
+  TrustToken__factory,
+} from 'contracts'
+
+use(solidity)
+
+describe('CollateralVault', () => {
+  let owner: Wallet
+
+  let tru: TrustToken
+  let borrowingMutex: BorrowingMutex
+  let lineOfCreditAgency: LineOfCreditAgency
+  let liquidator: Liquidator2
+
+  let collateralVault: CollateralVault
+
+  beforeEachWithFixture(async (wallets) => {
+    [owner] = wallets
+    const deployContract = setupDeploy(owner)
+
+    tru = await deployContract(TrustToken__factory)
+    borrowingMutex = await deployContract(BorrowingMutex__factory)
+    lineOfCreditAgency = await deployContract(LineOfCreditAgency__factory)
+    liquidator = await deployContract(Liquidator2__factory)
```

Comment:
Maybe let's do it with setupTruefi from the start?

Reply:
Yeah, good point. Otherwise all parallel follow-up PRs would need to duplicate work in switching to setupTrueFi2().